presto-docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -554,6 +554,46 @@ Drop a schema::
 
     DROP SCHEMA iceberg.web
 
+Register table
+^^^^^^^^^^^^
+
+Iceberg tables for which table data and metadata already exist in the
+file system can be registered with the catalog using the ``register_table``
+procedure on the catalog's ``system`` schema by supplying the target schema,
+desired table name, and the location of the table data and metadata::
```

Comment:
Are we assuming the table and metadata are at the same location?

Reply:
Great question! As of now, it does assume metadata and data are at the same location, even though that isn't necessarily a rule. Thinking I'll change the implementation slightly to check the fourth parameter for a fully qualified URI, and use that to locate the metadata. If the URI is relative, however, we'll stick with the assumption that it's co-located with the `data` folder. Thoughts?

Reply:
As I was (finally) addressing this comment, I believe that my previous comment was misguided. Since the table data is not accessed in a simple register situation, only the metadata location needs to be supplied. If the metadata and the data are not co-located, register still works as intended (there is a test to confirm as well). I've clarified modified the docs accordingly to indicate that only the metadata location is relevant to this procedure.